.github/workflows/build-publish.yml

```diff
@@ -45,9 +45,13 @@ jobs:
         platform:
           - 'manylinux1_x86_64'
           - 'manylinux1_i686'
+          - 'manylinux2014_aarch64'
     name: Build a ${{ matrix.platform }} for ${{ matrix.python_tag }}
     steps:
       - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
```

Comment:
I am assuming you know this better than me, but why is this necessary? Is there some documentation we can link in a comment here that explains why qemu is necessary to run the manylinux2014_aarch64 docker container?

Reply:
Github action provides x86_64 builders and not AArch64. Qemu is used to enable the execution of different multi-architecture containers. If we don’t use qemu, it will fail to run the AArch64 binaries on x86_64 machines.
If you have any further doubts please refer the [Link](https://github.com/multiarch/qemu-user-static)